### DISCUSSION TITLE
Can't join any servers

### DISCUSSION THREAD

**Discussion by Solwithabat** [ORelio/Minecraft-Console-Client#2143] [Q&A]
When I try and join a server I get this: Happens with servers other than hypixel.

![image](https://user-images.githubusercontent.com/112495177/187463337-4326fb19-88f9-44dc-9405-caadfa47ea27.png)

**Comment by milutinke** [comment]
Which version are you running?

**Reply by Solwithabat** [reply]
Minecraft Console Client v1.19 - for MC 1.4.6 to 1.19 - Github.com/MCCTeam on minecraft version 1.8.9 but it wasnt working on 1.19 either

**Reply by milutinke** [reply]
I mean the build version.
Also, can you enable debug messages and post screenshot(s) with them enabled?

**Reply by Solwithabat** [reply]
![debug1](https://user-images.githubusercontent.com/112495177/187550473-8ca69065-074b-4ce0-96c3-dc4e4c738980.JPG)
![debug2](https://user-images.githubusercontent.com/112495177/187550509-6ed47649-38a8-4e29-b522-f483d02b7556.JPG)
![debug3](https://user-images.githubusercontent.com/112495177/187550518-11df3cdd-be07-4eb8-8e07-a0f6c47e2b1c.JPG)


I'm not entirely sure what my build version is but i believe 
20220830-35 // https://github.com/MCCTeam/Minecraft-Console-Client/releases/tag/20220830-35 // on windows. Sorry if this wrong and i am being an idiot

**Reply by milutinke** [reply]
The issue occurs on 1.8.9, while 1.19 works fine.
I'll debug the issue when I catch some time, in the meantime, you can use a newer version, also try out few builds if you have time, and tell me of one works, maybe we could locate the bug that way.

**Comment by BruceChenQAQ** [comment]
Should be fixed in #2124
